src/components/home/modals/AddToken.vue

```diff
@@ -0,0 +1,172 @@
+              Token does not exist!
+            </b-form-invalid-feedback>
+            <b-button variant="warning"
+                      class="btn-o"
+                      block
+                      size="lg"
+                      :disabled="isAddable"
+                      @click="addModal">Add
+            </b-button>
+          </b-form-group>
+        </b-container>
+      </b-tab>
+    </b-tabs>
+  </b-modal>
+</template>
+
+<script>
+import Vue from 'vue'
+import { NODE_IP } from '../../../constants.js'
+import bus from '../../../assets/bus'
+export default {
+    name: 'AddToken',
+    data() {
+        return {
+            tokens: {},
+            tokenId: '',
+            init: false,
+            responseErr: false,
+            sendFlag: false
+        }
+    },
+    watch: {
+        tokenId() {
+            this.responseErr = false
+        }
+    },
+    computed: {
+        seedAddress() {
+            if (Vue.ls.get('address')) {
+                return Vue.ls.get('address')
+            }
+        },
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.seedAddress))
+        },
+        isAddable() {
+            return !this.tokenId.length > 0
+        }
+    },
+
+    methods: {
+        closeModal() {
+            this.init = false
+            this.responseErr = false
+            this.tokenId = ''
+            this.tokens = {}
+            this.$refs.addTokenModal.hide()
+        },
+        setUsrLocalStorage(fieldname, value) {
+            Vue.set(this.userInfo, fieldname, value)
+            window.localStorage.setItem(this.seedAddress, JSON.stringify(this.userInfo))
+        },
+        addModal() {
+            this.init = true
+            this.tokens = {}
+            var tmpUserInfo = JSON.parse(window.localStorage.getItem(this.seedAddress))
+            if (tmpUserInfo && tmpUserInfo.tokens) {
+                this.tokens = JSON.parse(tmpUserInfo.tokens)
+            }
+            const url = NODE_IP + '/contract/tokenInfo/' + this.tokenId
+            this.$http.get(url).then(response => {
+                this.responseErr = false
+                Vue.set(this.tokens, this.tokenId, JSON.parse(JSON.stringify(this.tokenId)))
+                this.setUsrLocalStorage('tokens', JSON.stringify(this.tokens))
+                var usInfo = JSON.parse(window.localStorage.getItem(this.seedAddress))
+                console.log(usInfo.tokens)
+                this.sendFlag = true
+                bus.$emit('sendFlag', this.sendFlag)
+                this.sendFlag = false
+                this.$refs.addTokenModal.hide()
+            }, respError => {
+                this.responseErr = true
+            })
+        },
+        isValidToken() {
+            if (!this.init || this.tokenId.length === 0 || this.responseErr === false) {
+                return void 0
+            }
+            return !this.responseErr
```

Comment:
if this.responseErr == false, it will be return void 0 as line 116, and why line 118?